CommandLine/protos/resources/Object.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto2";
+package buffers.resources;
+
+import "options.proto";
+import "resources/Event.proto";
+
+message Object {
+  optional string name = 1;
+  optional int32 id = 2 [(gmx) = "GMX_DEPRECATED"];
+
+  optional string parent_name = 3 [(gmx) = "parentName"];
+  optional string sprite_name = 4 [(gmx) = "spriteName"];
+  optional string mask_name = 5 [(gmx) = "maskName"];
+
+  optional int32 depth = 6;
+  optional bool solid = 7;
+  optional bool visible = 8;
+  optional bool persistent = 9;
+
+  repeated Event events = 10 [(gmx) = "events/event"];
+
+  enum PhysicsShape { CIRCLE = 0; BOX = 1; SHAPE = 2; }
+
+  message PhysicsShapePoint {
+    optional int32 x = 1 [(gmx) = "GMX_SPLIT/0"];
+    optional int32 y = 2 [(gmx) = "GMX_SPLIT/1"];
+  }
+
+  optional bool use_physics = 11 [(gmx) = "PhysicsObject"];

```

Comment:
Do we have the ability to extract these into a nested message? I'd really like to.

Reply:
Not currently. The gmx parser is set to go up xml elements along with the protobufs. It can go down the ladder in the xml but not up.

Reply:
Lol maybe we just add support for "../" then? Does XPATH have a standard for that?